### PR TITLE
Refactored spectron tests

### PR DESCRIPTION
**Pull request by KiranNiranjan**
@VikasShashidhar Please review. Thanks

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -79,6 +80,7 @@
     "eslint-plugin-jsx-a11y": "^4.0.0",
     "eslint-plugin-react": "^6.10.0",
     "jest": "^19.0.2",
+    "ncp": "^2.0.0",
```
Comment (VikasShashidhar) [inline package.json]: Is there no other way we can do this? I mean, copying the config file to the required path. If there is no other way, then I'll merge this.

File: tests/bringToFront.test.js [hunk 2]
```diff
@@ -1,56 +1,83 @@
-const Application = require('./spectronSetup');
-
 describe('Tests for Bring to front', () => {
 
     let originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
-    jasmine.DEFAULT_TIMEOUT_INTERVAL = 60000;
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 90000;
```
Comment (VikasShashidhar) [inline tests/bringToFront.test.js]: Can we get these from a method in spectronSetup instead of hardcoding it here?

File: package.json (hunk 1 above)
Comment (KiranNiranjan) [inline package.json]: This package is used because it has a ```CLI``` option which works both on windows and mac

**Comment by KiranNiranjan** [comment]
@VikasShashidhar made changes. Please review. Thanks

File: package.json (hunk 1 above)
Comment (VikasShashidhar) [inline package.json]: Ok 👍

**Comment by VikasShashidhar** [comment]
LGTM 👍

**Pull request merged**